sdk/search/search-documents/test/odata.spec.ts

```diff
@@ -30,6 +30,73 @@ describe("odata", () => {
     assert.strictEqual(result, "search.ismatch('you''re', 'Description')");
   });
 
+  it("no arguments", () => {
+    assert.strictEqual(odata`Foo eq 2`, "Foo eq 2");
+  });
+
+  it("one argument", () => {
+    assert.strictEqual(odata`Foo eq ${2}`, "Foo eq 2");
+  });
+
+  it("many arguments", () => {
+    assert.strictEqual(odata`Foo eq ${2} and Bar eq ${3}`, "Foo eq 2 and Bar eq 3");
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4"
+    );
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4} and Qux eq ${5}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4 and Qux eq 5"
+    );
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4} and Qux eq ${5} and Quux eq ${6}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4 and Qux eq 5 and Quux eq 6"
+    );
```

Comment:
do we need to bother with so many cases? It feels like we're just testing that string templates work at some point rather than our code.

Reply:
:-) Removed all but one.